src/utils/whitelistApis.ts

```diff
@@ -4645,6 +4659,8 @@ export const API_LIST = {
             '/proxies/v8/mentoring/v1/entity-type/read',
             '/proxies/v8/mentoring/v1/entity/list',
             '/proxies/v8/data/v2/system/settings/get/:id',
-            '/proxies/v8/mentoring/v1/sessions/update'
+            '/proxies/v8/mentoring/v1/sessions/update',
+            '/proxies/v8/halloffame/v1/mdoleaderboard',
+            '/proxies/v8/halloffame/v1/userleaderboard/:id'
```

Comment:
Please add '/proxies/v8/halloffame/v1/userleaderboard/:id',
comma at the last

Reply:
added